public/docs/ts/latest/guide/glossary.jade

```diff
@@ -0,0 +1,445 @@
+
+ :markdown
+  ## Directive
+.l-sub-section
+  :markdown
+    An Angular class responsible for creating, re-shaping, and interacting with HTML elements
+    in the browser DOM. Directives are Angular's most fundamental feature.
+    
+    A Directive is almost always associated with an HTML element or attribute.
+    We often refer to such an element or attribute as the directive itself. 
+    When Angular finds a directive in an HTML template, 
+    it creates the matching directive class instance 
+    and gives that instance control over that portion of the browser DOM.
+    
+    Developers can invent custom HTML markup (e.g., `<my-directive>`) to
+    associate with their custom directives. They add this custom markup to HTML templates
+    as if they were writing native HTML. In this way, directives become extensions of 
+    HTML itself.  
+    
+    Directives fall into one of three categories:
+    
+    1. [Components](#component) that combine application logic with an HTML template to
+    render application [views]. Components are usually represented as HTML elements. 
+    They are the building blocks of an Angular application and the 
+    developer can expect to write a lot of them. 
+    
+    1. [Attribute Directives](attribute-directive) that can listen to and modify the behavior of
+    other HTML elements, attributes, properties, and components. They are usually represented
+    as HTML attributes, hence the name.
+    
+    1. [Structural Directives](#structural-directive), a directive responsible for 
+    shaping or re-shaping HTML layout, typically by adding, removing, or manipulating
+    elements and their children.
+ 
+.l-main-section
+:markdown
+  ## ECMAScript
+.l-sub-section
+  :markdown
+    The [official JavaScript language specification](https://en.wikipedia.org/wiki/ECMAScript).
+    
+    The lastest released version of JavaScript is  
+    [ECMAScript 2015](http://www.ecma-international.org/ecma-262/6.0/)
+    (AKA "ES2015" or "ES6") and many Angular 2 developers will write their applications 
+    either in this version of the language or a dialect that strives to be 
+    compatible with it such as [TypeScript](#typesScript).
+    
+    Most modern browsers today only support the prior "ECMAScript 5" (AKA ES5) standard.
+    Applications written in ES2015 or one of its dialects must be "[transpiled](transpile)"
+    to ES5 JavaScript.
+    
+    Angular 2 developers may choose to write in ES5 directly.
+:markdown
+  ## ECMAScript 2015
+.l-sub-section
+  :markdown
+    The lastest released version of JavaScript, 
+    [ECMAScript 2015](http://www.ecma-international.org/ecma-262/6.0/)
+    (AKA "ES2015" or "ES6") 
+:markdown
+  ## ES2015
+.l-sub-section
+  :markdown
+    Short hand for "[ECMAScript 2015](#ecmascript=2015)". 
+:markdown
+  ## ES6
+.l-sub-section
+  :markdown
+    Short hand for "[ECMAScript 2015](#ecmascript=2015)". 
```

Comment:
We can update as the term ES6 finally disappears. There are still a great many references to ES6